### ISSUE TITLE
Panic while verifying rewardValidatorTx

### ISSUE THREAD

**Issue by danlaine** [ava-labs/gecko#148]
**Describe the bug**
P-Chain panics with:
```
FATAL[05-08|15:01:12] <chain 11111111111111111111111111111111LpoYY> /utils/logging/log.go#247: Panicing due to:
runtime error: invalid memory address or nil pointer dereference
From:
goroutine 86 [running]:
github.com/ava-labs/gecko/utils/logging.Stacktrace.String(0x450b00, 0xc000316c01, 0x7f94a0736e28)
	/home/fawkes/Downloads/gecko/utils/logging/stack.go:20 +0x71
fmt.(*pp).handleMethods(0xc000270dd0, 0x73, 0x6400000001)
	/usr/local/go/src/fmt/print.go:630 +0x302
fmt.(*pp).printArg(0xc000270dd0, 0xdcf560, 0xc001c05c5c, 0x73)
	/usr/local/go/src/fmt/print.go:713 +0x206
fmt.(*pp).doPrintf(0xc000270dd0, 0xee8a91, 0x1c, 0xc0003171b8, 0x2, 0x2)
	/usr/local/go/src/fmt/print.go:1030 +0x15b
fmt.Sprintf(0xee8a91, 0x1c, 0xc0003171b8, 0x2, 0x2, 0xc001c02c00, 0x35)
	/usr/local/go/src/fmt/print.go:219 +0x66
github.com/ava-labs/gecko/utils/logging.(*Log).format(0xc000510900, 0x1, 0xee8a91, 0x1c, 0xc0003171b8, 0x2, 0x2, 0x1, 0x48)
	/home/fawkes/Downloads/gecko/utils/logging/log.go:165 +0x11a
github.com/ava-labs/gecko/utils/logging.(*Log).log(0xc000510900, 0x1, 0xee8a91, 0x1c, 0xc0003171b8, 0x2, 0x2)
	/home/fawkes/Downloads/gecko/utils/logging/log.go:138 +0x11f
github.com/ava-labs/gecko/utils/logging.(*Log).Fatal(...)
	/home/fawkes/Downloads/gecko/utils/logging/log.go:180
github.com/ava-labs/gecko/utils/logging.(*Log).StopOnPanic(0xc000510900)
	/home/fawkes/Downloads/gecko/utils/logging/log.go:247 +0xf4
panic(0xd8e740, 0x1992b00)
	/usr/local/go/src/runtime/panic.go:679 +0x1b2
sync.(*RWMutex).RLock(...)
	/usr/local/go/src/sync/rwmutex.go:48
github.com/ava-labs/gecko/database/versiondb.(*Database).Has(0x0, 0xc00064e820, 0x20, 0x20, 0xeb512b581bb1f600, 0x0, 0x0)
	/home/fawkes/Downloads/gecko/database/versiondb/db.go:40 +0x42
github.com/ava-labs/gecko/vms/components/state.(*state).Has(0xc0004b8cb0, 0x11608c0, 0x0, 0x1, 0xc001c406c0, 0x28, 0x28, 0x28)
	/home/fawkes/Downloads/gecko/vms/components/state/state.go:118 +0x75
github.com/ava-labs/gecko/vms/platformvm.(*VM).getCurrentValidators(0xc00053b8c0, 0x11608c0, 0x0, 0x1b0ba20, 0xc0003174e8, 0x4257ce, 0xc5bb96)
	/home/fawkes/Downloads/gecko/vms/platformvm/state.go:33 +0xbb
github.com/ava-labs/gecko/vms/platformvm.(*rewardValidatorTx).SemanticVerify(0xc0008b25a0, 0x11608c0, 0x0, 0x7f94a85e89e8, 0xc0003cc1e0, 0x1, 0xd69ce0, 0xc0003d4960, 0xc0000c6548)
	/home/fawkes/Downloads/gecko/vms/platformvm/reward_validator_tx.go:69 +0x96
github.com/ava-labs/gecko/vms/platformvm.(*ProposalBlock).Verify(0xc000f6ba40, 0xc0003177e0, 0x1)
	/home/fawkes/Downloads/gecko/vms/platformvm/proposal_block.go:106 +0xa2
github.com/ava-labs/gecko/snow/engine/snowman.(*Transitive).deliver(0xc0000d6840, 0x115ae20, 0xc000f6ba40)
	/home/fawkes/Downloads/gecko/snow/engine/snowman/transitive.go:350 +0x10b
github.com/ava-labs/gecko/snow/engine/snowman.(*issuer).Update(...)
	/home/fawkes/Downloads/gecko/snow/engine/snowman/issuer.go:43
github.com/ava-labs/gecko/snow/engine/snowman.(*issuer).Fulfill(0xc0003d4690, 0xc001bf1dc0)
	/home/fawkes/Downloads/gecko/snow/engine/snowman/issuer.go:22 +0xa8
github.com/ava-labs/gecko/snow/events.(*Blocker).Fulfill(0xc0000d6ab8, 0xc001bf1dc0)
	/home/fawkes/Downloads/gecko/snow/events/blocker.go:32 +0x161
github.com/ava-labs/gecko/snow/engine/snowman.(*Transitive).deliver(0xc0000d6840, 0x115ae20, 0xc000f6bb90)
	/home/fawkes/Downloads/gecko/snow/engine/snowman/transitive.go:384 +0x36b
github.com/ava-labs/gecko/snow/engine/snowman.(*issuer).Update(0xc0003d4c30)
	/home/fawkes/Downloads/gecko/snow/engine/snowman/issuer.go:43 +0x61
github.com/ava-labs/gecko/snow/events.(*Blocker).Register(0xc0000d6ab8, 0x1151240, 0xc0003d4c30)
	/home/fawkes/Downloads/gecko/snow/events/blocker.go:59 +0x289
github.com/ava-labs/gecko/snow/engine/snowman.(*Transitive).insert(0xc0000d6840, 0x115ae20, 0xc000f6bb90)
	/home/fawkes/Downloads/gecko/snow/engine/snowman/transitive.go:286 +0x183
github.com/ava-labs/gecko/snow/engine/snowman.(*Transitive).insertFrom(0xc0000d6840, 0xc000362440, 0x115ae20, 0xc000f6bb90, 0x0)
	/home/fawkes/Downloads/gecko/snow/engine/snowman/transitive.go:245 +0xc2
github.com/ava-labs/gecko/snow/engine/snowman.(*Transitive).reinsertFrom(0xc0000d6840, 0xc000362440, 0xc001ab9e80, 0xc00061d400)
	/home/fawkes/Downloads/gecko/snow/engine/snowman/transitive.go:239 +0x93
github.com/ava-labs/gecko/snow/engine/snowman.(*Transitive).Chits(0xc0000d6840, 0xc000362440, 0x3, 0xc0008c5530)
	/home/fawkes/Downloads/gecko/snow/engine/snowman/transitive.go:176 +0x304
github.com/ava-labs/gecko/snow/networking/handler.(*Handler).dispatchMsg(0xc0004cf860, 0xc, 0xc000362440, 0x3, 0x0, 0x0, 0x0, 0x0, 0xc0008c5530, 0x0, ...)
	/home/fawkes/Downloads/gecko/snow/networking/handler/handler.go:91 +0x581
github.com/ava-labs/gecko/snow/networking/handler.(*Handler).Dispatch(0xc0004cf860)
	/home/fawkes/Downloads/gecko/snow/networking/handler/handler.go:43 +0x2b3
github.com/ava-labs/gecko/utils/logging.(*Log).RecoverAndPanic(0xc000510900, 0xc0004b9b00)
	/home/fawkes/Downloads/gecko/utils/logging/log.go:254 +0x57
created by github.com/ava-labs/gecko/chains.(*manager).createSnowmanChain
	/home/fawkes/Downloads/gecko/chains/manager.go:524 +0x740
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x9996b2]
goroutine 86 [running]:
github.com/ava-labs/gecko/utils/logging.(*Log).StopOnPanic(0xc000510900)
	/home/fawkes/Downloads/gecko/utils/logging/log.go:249 +0x11d
```

**To Reproduce**
Run a node. Otherwise, unknown.

**Expected behavior**
No panic

**Operating System**
Ubuntu 18.04

**Additional context**

By submitting this issue I agree to the [Terms and Conditions](https://files.avalabs.org/legal/bug_hunt_q2_2020_terms.pdf) of the [Developer Accelerator Program](https://avalabs.org/dap).

**Comment by StephenButtolph** [comment]
Was this reported on a node that had updated to #119? It's possible this same stacktrace could be caused by something else, but I just want to make sure that we aren't searching for an already resolved bug.

**Comment by danlaine** [comment]
@StephenButtolph Not sure...seems likely that this is from the bug resolved in #119. Closing this issue, and will re-open if someone reports it again.
